projects/core/src/i18n/config/i18n-config.ts

```diff
@@ -23,18 +23,41 @@ export abstract class I18nConfig {
      * Configuration for lazy loading of translation files.
      * For eager loading of translations please use config option `i18n.resources`
      */
-
     backend?: {
       /**
        * The path to JSON translations. It should contain placeholders:
        * - `{{lng}}` for language
        * - `{{ns}}` for the name of chunk.
        *
        * Example:
-       * `assets/i18n-assets/{{lng}}/{{ns}}.json`
+       * ```ts
+       * { loadPath: 'assets/i18n-assets/{{lng}}/{{ns}}.json' }
+       * ```
+       *
+       * **CAUTION: It's recommended to use the `loader` config property instead, for SSR performance reasons
+       * (for more, see https://github.com/SAP/spartacus/issues/13460).**
+       *
+       *
+       * If you really want to use this config, you'll need to additionally provide `I18nextHttpBackendService`
+       * for `I18nextBackendService`:
+       * ```ts
+       * { provide: I18nextBackendService, useExisting: I18nextHttpBackendService }
+       * ```
        */
-
       loadPath?: string;
```

Comment:
shall we deprecate it? 
do we know any use cases when would customers choose `loadPath` over the more performant `loader`?